co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstanceManager.java

```diff
@@ -91,25 +101,30 @@ public void onNewRegistration(InfrastructureRegistrationMessage registration) {
      * 
      */
     private void newInfrastructureInstance(String infrastructureId, InetAddress rxMessageIpAddress, int rxMessagePort,
-            int timeSyncPort, CartesianPoint location) {
+            int timeSyncPort, int simulatedInteractionPort, CartesianPoint location, List<Detector> sensors) {
         InfrastructureInstance tmp = new InfrastructureInstance(infrastructureId, rxMessageIpAddress, rxMessagePort,
-                timeSyncPort, location);
+                timeSyncPort, simulatedInteractionPort, location, sensors);
         try {
             tmp.bind();
-            log.info("New Infrastructure instance '{}' registered with Infrastructure Instance Manager.", infrastructureId);
+            log.info("New Infrastructure instance '{}' registered with Infrastructure Instance Manager.",
+                    infrastructureId);
         } catch (IOException e) {
             log.error("Failed to bind infrastructure instance with ID '{}' to its RX message socket: {}",
                     infrastructureId, e.getMessage());
             log.error("Stack trace:", e);
         }
+
         managedInstances.put(infrastructureId, tmp);
     }
 
     /**
-     * Callback to be invoked when CARMA Platform receives a V2X Message from the NS-3 simulation
+     * Callback to be invoked when CARMA Platform receives a V2X Message from the
+     * NS-3 simulation
+     * 
      * @param sourceAddr The V2X Message received
-     * @param txMsg The Host ID of the vehicle receiving the data
-     * @throws RuntimeException If the socket used to communicate with the platform experiences failure
+     * @param txMsg      The Host ID of the vehicle receiving the data
```

Comment:
The actual V2x message that received from platform

Reply:
I can edited this method comment as well but it is also not part of my PR. Looks like some formatting changes changed the comment format but there are not actual changes to the method.